test/integration/proposal-genericSchemeStandardBounties.ts

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
why this test  file is comment out ? @tibetsprague @dkent600 ?

Reply:
good question, I guess he never got it to work? or maybe never worked on it. probably we shouldn't have let this PR through without the tests... but we did this time

Reply:
ok. please open an issue to enable this test (and make it work:)) .